phy/plot/waveforms.py

```diff
@@ -283,16 +318,16 @@ def on_key_press(self, event):
         shift = 'Shift' in event.modifiers
 
         # Box scale.
-        if ctrl and key in self._arrows:
+        if ctrl and key in self._arrows + self._pm:
             coeff = 1.1
             u, v = self.box_scale
             if key == 'Left':
                 self.box_scale = (u / coeff, v)
             elif key == 'Right':
                 self.box_scale = (u * coeff, v)
-            elif key == 'Down':
+            elif key in ('Down', '-'):
                 self.box_scale = (u, v / coeff)
-            elif key == 'Up':
+            elif key in ('Up', '+'):
```

Comment:
can we make this `=` as well as `+`? because otherwise you have to press 'shift' to get to it on a QWERTY keyboard :/